File: CPP/Intro a la prog/ejercicio2.cpp
#include <iostream>
using namespace std;

char name[15]; float time; float salary;

int main(){
	cout << "Inserte su nombre." << endl;
	cin >> name;
	cout << "Inserte su tiempo de trabajo." << endl;
	cin >> time; 
	cout << "Inserte su salario." << endl;
	cin >> salary;
	cout << "El salario neto del empleado " << name << " es de " << (time*salary)*(90.0/100.0) << "Bs.";
	return 0;
}

File: Lua/ejercicio2.lua
print("Introduzca su nombre, el tiempo que trabajo, y el costo de sus horas de trabajo, separados por una linea.")
local name,time,salary = io.read(),tonumber(io.read()),tonumber(io.read())
print(("El empleado %s recibira un salario neto de %s con impuestos incluidos."):format(name,(time*salary)*0.9))